ansibullbot/triagers/plugins/component_matching.py

```diff
@@ -142,6 +145,14 @@ def get_component_match_facts(iw, component_matcher, valid_labels):
                 if u'/plugins/' in x:
                     cmeta[u'is_new_plugin'] = True
 
+    # is it a colleciton?
+    if [x for x in CM_MATCHES if x[u'repo_filename'].startswith(u'collection:')]:
+        cmeta[u'is_collection'] = True
+        cmeta[u'component_collection'] = []
+        cmeta[u'component_support'] = [u'community']
+        for comp in [x for x in CM_MATCHES if x[u'repo_filename'].startswith(u'collection:')]:
```

Comment:
Using a generator expression will prevent the creation of an unnecessary in-memory list:
```suggestion
        for comp in (x for x in CM_MATCHES if x[u'repo_filename'].startswith(u'collection:')):
```